### PR TITLE
Feature/parent filter

### PR DESCRIPTION
**Pull request by DavidOberholzer**
Added a parent id filter and bumped the core-shared version to include my required fix. Added tests for core shared list_entry here.

### CODE REVIEW

**Comment by AltusBarry** [comment]
+1

File: access_control/test/test_methods.py [hunk 1]
```diff
@@ -24,3 +24,42 @@ def test_get_or_create(self):
...
+            pass
+        cls.domain_1, created = get_or_create(models.Domain, name="Domain 1")
+        cls.domain_2, created = get_or_create(models.Domain, name="Domain 2")
+
+    def test_list_entry(self):
+        entries = list_entry(
+            models.Domain,
+            query={
+                "ids": {
+                    "id": self.domain_1.id
```
Comment (AltusBarry) [inline access_control/test/test_methods.py]: With the new change, wouldnt this need to test `ids:{"parent_id": id}` and `ids:{ "parent_id", [id, id]}`?

Comment (DavidOberholzer) [reply access_control/test/test_methods.py]: Good Idea to test more than just the core-shared update. will do

**Comment by AltusBarry** [comment]
+1

**Pull request merged**